client/src/components/AuditAdmin/Progress/JurisdictionDetail.tsx

```diff
@@ -125,6 +128,16 @@ const JurisdictionDetail: React.FC<IJurisdictionDetailProps> = ({
             auditSettings={auditSettings}
           />
         )}
+        {lastLoginActivity && (
+          <>
+            <H5>Last Login</H5>
+            <span>
+              {`${lastLoginActivity.user?.key || 'Unknown'} at ${new Date(
```

Comment:
Is it possible for `user` to be undefined? It feels like if a user logged in, there should be a user defined...

Reply:
`IActivity` ([src](https://github.com/votingworks/arlo/pull/2061/files#diff-79e5cec27ff845c276566983488e2e282ca8a908829e0575d3fad9f14ec48ef1R10)) doesn't have subtypes for different activities so `activity.user` is nullable, but you're right we'll never hit this fallback in practice. Removed the fallback in [ec442b4](https://github.com/votingworks/arlo/pull/2061/commits/ec442b44d0d0bde29574ee57ebbaf9a5209aa770)

We could extend these types in a follow up PR if that level of granularity is helpful in other places.